docs/authorization_userguide.md

```diff
@@ -59,8 +59,20 @@ Currently, Supported statements are as follows:
 | SHOW COLUMNS    | ❌ | ❌ |
 | DESCRIBE COLUMN | ❌ | ❌ |
 
+## Compatibility with Data Source API
+
+We support authorization check in both read and write, but we don't recommend to use read.
```

Comment:
```suggestion
We support authorization check in both read and write, but we recommend using spark SQL instead of Datasource API to read data from TiDB
```

Reply:
This may be clearer